test/examples/mixed.docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+services:
+  collector:
+    # hopefully 0.27.0 will include polling exporter
+    # image: quay.io/signalfx/splunk-otel-collector:0.27.0
+    image: ghcr.io/splunk-o11y-gdi-bot/splunk-otel-collector-dev:polling
+    environment:
+      - SPLUNK_CONFIG=/etc/otel/config.yml
+    volumes:
+      - ./collector.config.yml:/etc/otel/config.yml
+    # ports:
```

Comment:
Remove commented out stuff?

Reply:
Sure. I'll leave the bottom ones since they are "functional" and not trivial to repro by heart if you think that's fine.